Maquetado/scss/footer.scss

```diff
@@ -0,0 +1,33 @@
+$image-logo: './assets/wolox_logo_white.svg';
+
+.footer {
+  background-color: $blue;
+  display: flex;
+  flex-flow: column wrap;
+  align-items: center;
+  justify-content: space-around;
+  margin-bottom: 10px;
+  padding: 20px;
+
+  .footer-slogan {
+    font-size: 15px;
+    color: $white;
+    margin-bottom: 20px;
+  }
+
+  .footer-img {
+    background-image: url($image-logo);
+    background-repeat: no-repeat;
+    margin-bottom: 20px;
+    object-fit: contain;
+    padding: 30px;
+    width: 400px;
```

Comment:
setting a fixed width this big will break the layout in smaller screens. use:
```
width: 100%;
max-width: 400px;
```
to fix this.